packages/bpk-component-popover/src/BpkPopover.tsx

```diff
@@ -154,9 +154,20 @@ const BpkPopover = ({
     dismiss,
   ]);
 
+  const targetProps = target.props;
```

Comment:
I'd say I wouldn't extract all the props to a variable here as it will mean we aren't spreading any of the others anywhere else, so just either set this to be

`const targetClick = target.props.onClick` or on line 164 just simply call it as `if (target.props.onClick) {`

Reply:
Thank you very much for your suggestion. 
The target is defined as `ReactElement<any>`, and with the any type, directly using `target.props.onClick` would result in a TypeScript type error. 

However, defining `const targetClick = target.props.onClick` works fine. 

The code has been updated. Thanks.